core/src/main/java/org/apache/jsieve/SieveFactory.java

```diff
@@ -107,16 +106,16 @@ public Node parse(InputStream inputStream) throws ParseException {
             node.jjtAccept(visitor, null);
             return node;
         } catch (ParseException ex) {
-            if (log.isErrorEnabled())
-                log.error("Parse failed. Reason: " + ex.getMessage());
-            if (log.isDebugEnabled())
-                log.debug("Parse failed.", ex);
+            if (LOGGER.isErrorEnabled())
+                LOGGER.error("Parse failed. Reason: " + ex.getMessage());
```

Comment:
Warning: you swallow stacktrace!

Let's keep a simple error log with the stacktrace here